index.html

```diff
@@ -0,0 +1,533 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/5b1eeceadd.js" crossorigin="anonymous"></script>
+  <title>TNW-Website-Clone</title>
+  <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+  <header>
+    <nav class="nav">
+      <div class="nav-logo">
+        <a href="index.html"><img src="./images/tnw-red.svg" alt=""></a>
+        <a href=""><i class="fas fa-bars bar fa-2x"></i></a>
+      </div>
```

Comment:
- [ ] In medium view the TWN logo appears at the leftmost part but the navigation menu overlaps it and there is a redundant empty space on the right side of the navigation menu. Please kindly fix it.

![image](https://user-images.githubusercontent.com/60184961/93028789-e171ba00-f61e-11ea-8820-4788d1f5479a.png)